guides/v2.2/config-guide/redis/redis-session.md

```diff
@@ -53,17 +43,21 @@ Known issue with strings of more than 64KB.|2048|
 |session-save-redis-disable-locking|disable_locking|Disable session locking entirely.|0 (false)|
 |session-save-redis-min-lifetime|min_lifetime|Minimum session lifetime, in seconds.|60|
 |session-save-redis-max-lifetime|max_lifetime|Maximum session lifetime, in seconds.|2592000 (720 hours)|
+{:style="table-layout:auto;"}
 
 ### Example command
 
 The following example sets Redis as the session data store, sets the host to `127.0.0.1`, sets the log level to 3, and sets the database number to 2. All other parameters are set to the default value.
 
+```bash
 `bin/magento setup:config:set --session-save=redis --session-save-redis-host=127.0.0.1 --session-save-redis-log-level=3 --session-save-redis-db=2`
```

Comment:
Remove the single backticks surrounding the command.

Reply:
Done